homeassistant/components/broadlink/remote.py

```diff
@@ -373,7 +373,7 @@ async def _async_learn_rf_command(self, command):
             start_time = dt_util.utcnow()
             while (dt_util.utcnow() - start_time) < LEARNING_TIMEOUT:
                 await asyncio.sleep(1)
-                found = await device.async_request(device.api.check_frequency)
+                found = await device.async_request(device.api.check_frequency)[0]
```

Comment:
Looks like this is for a breaking change in the lib
https://github.com/mjg59/python-broadlink/compare/0.18.3...0.19.0#diff-c39b1968c7408ed7de5265614be5eaf7eff3f8e4b296b5399593b69deefd4f05R91